File: guess.c
#include <stdio.h>
#include <stdlib.h>
int main()
{
    int SecretNo = 5;
    int guess = 0;
    int guessCount = 0;
    int guessLimit = 3;
    int outofG = 0;

    while(guess != SecretNo && outofG == 0)
    {
        if (guessCount < guessLimit)
        {
            printf("Enter your guess: ");
        scanf("%d", &guess);
        guessCount++;
        }
        else
            outofG = 1 ;
    }
    if(outofG == 0)
    printf("You Win\n");
    else
        printf("You are out of Guesses\n");
return 0;
}

File: guess.py
secret_word = "Ankit"
Guess_count = 3
guess = ""
while guess != secret_word and Guess_count < 1
    guess = input("Your Guess: ")
    if guess == secret_word:
         print("Correct choice")
    else
    print("Wrong guess")
    Guess_count -= 1
    print("Guesses remaining: " + Guess_count)
if guess == secret_word:
    print("Correct choice")


